src/db.rs

```diff
@@ -198,6 +198,10 @@ impl Database {
                 debug!("output:\n{}", chunk);
             }
             if !column_names.is_empty() && !output.is_empty() {
+                if output.len() > 1 {
+                    let chunks: Vec<DataChunk> = output.drain(1..).collect();
+                    output[0].stack(chunks);
+                }
```

Comment:
Would it be better to merge chunks in `main.rs` (e.g., change `print_chunk(chunk: &DataChunk)` to print chunks) instead of `run`? `run` returns `Vec<DataChunk>`. We'd better not change it?

Reply:
User may submit multiple statments to client at once, 
For example, the input is
```sql
create table t; insert into x values (1); insert into x values (2); select * from x; insert into x values (3333);
```
then there might be multiple statements in a `run`, how to handle this case in the print_chunks? I think we need to add some complex logic.

Reply:
Oh this is a good catch! So each element of the returned `Vec` of `run` is the results for one command. I think it makes sense.